cluster/images/etcd/rollback/rollback.go

```diff
@@ -26,6 +26,8 @@ import (
 	"strings"
```

Comment:
```
W1215 09:36:03.180] wrote BUILD for "/go/src/k8s.io/kubernetes/third_party/forked/etcd237/pkg/fileutil"
W1215 09:36:03.195] wrote BUILD for "/go/src/k8s.io/kubernetes/third_party/forked/etcd237/wal"
W1215 09:36:03.196] extract err: cannot find package "github.com/coreos/etcd237/Godeps/_workspace/src/github.com/gogo/protobuf/proto" in any of:
W1215 09:36:03.196] 	/go/src/k8s.io/kubernetes/vendor/github.com/coreos/etcd237/Godeps/_workspace/src/github.com/gogo/protobuf/proto (vendor tree)
W1215 09:36:03.196] 	/usr/local/go/src/github.com/coreos/etcd237/Godeps/_workspace/src/github.com/gogo/protobuf/proto (from $GOROOT)
W1215 09:36:03.196] 	/go/src/github.com/coreos/etcd237/Godeps/_workspace/src/github.com/gogo/protobuf/proto (from $GOPATH)
W1215 09:36:03.197] wrote BUILD for "/go/src/k8s.io/kubernetes/third_party/forked/etcd237/wal/walpb"
```

This says two things.
1. this package won't compile because [this import](https://github.com/kubernetes/kubernetes/blob/d26bf3195007534453c2324ef18e95694b5a9a46/third_party/forked/etcd237/wal/walpb/record.pb.go#L20) is not in our gopath. I can verify this by checking out your branch and running `go install ./third_party/forked/etcd237/...`. I get a compilation error.
2. `BUILD` files need to be update in `third_party/forked/etcd237/wal`, `third_party/forked/etcd237/pkg/fileutil`, and `third_party/forked/etcd237/wal/walpb` which makes sense because these are brand new packges. You can do this by rebasing this PR onto latest master and running `./hack/update-bazel.sh` then commiting the changes.